tests/foreman/api/test_activationkey.py

```diff
@@ -173,8 +173,12 @@ def test_positive_update_limited_host(max_host):
     :parametrized: yes
     """
     # unlimited_hosts defaults to True.
-    act_key = entities.ActivationKey().create()
-    want = {'max_hosts': max_host, 'unlimited_hosts': False}
+    act_key = target_sat.api.ActivationKey().create()
+    want = {
+        'max_hosts': max_host,
+        'unlimited_hosts': False,
+        'organization_id': act_key.organization_id,
```

Comment:
Can you keep this consistent?
```suggestion
        'organization_id': act_key.organization.id
```

Reply:
act_key does contain organization.id but the ActivationKey entity still requires ``organization_id`` as a param.  This was a typo, ill update to the new var

Reply:
ack_key is nailgun object for AK which has organization object for org, so you could do `act_key.organization.id` to get the ID
```
In [5]: act_key
Out[5]: robottelo.hosts.DecClass(auto_attach=True, content_view=None, description=None, environment=None, host_collection=[], max_hosts=None, name='FjdRZMrLWTk', organization=nailgun.entities.Organization(id=3), purpose_usage=None, purpose_role=None, purpose_addons=[], release_version=None, service_level=None, unlimited_hosts=True, id=1)
```

Reply:
I think either one works here. I had to add organization_id to the ActivationKey entity already. Might as well use it in the call since its there

Reply:
yes, both can be used, but why adding one more organization_id param when we already have organization param for same thing :)

Reply:
There's organization field already present in ActivationKey entity, so not sure why we need explicit organization_id field when it can find with `organization.id`